charter.md

```diff
@@ -85,15 +93,123 @@ are replacing, regardless of the length of that remainder.
 
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
```

Comment:
Can we say "Currently this translates to" instead of "Example"?

Reply:
Leaving as-is, as this section matches the other vote `Example`s elsewhere in the charter.

Reply:
...additionally, I added the `Composition`, so someone could derive the correct numbers based on the fixed membership of the committee.

The examples just happen to be based on current fixed composition of the committee.
Should the composition of the committee change in the future, any instances of "currently" would have to be corrected.